### DISCUSSION TITLE
Linebreaks are not being respected in the backstory field in user profiles

### DISCUSSION THREAD

**Discussion by smallsco** [BattlesnakeOfficial/feedback#61] [Bug Reports]
The "What is your backstory?" field, when updating your profile, is a multi-line text area, which gives the impression that you can use linebreaks in the text:

![image](https://user-images.githubusercontent.com/1906551/122453106-d7d70300-cf5e-11eb-80ba-7780efccd12d.png)

However, when viewing your profile, the linebreaks are stripped, making it difficult to read:

![image](https://user-images.githubusercontent.com/1906551/122453224-fa691c00-cf5e-11eb-8827-768f098ad5fc.png)

This feels like a bug - ideally the linebreaks should be displayed when viewing a profile. Or if it's intended then you shouldn't be allowed to enter them at all when updating your profile (or at least put in a note saying they'll be stripped or something)

(Bonus points if you can make URLs in the backstory clickable, or add a separate profile section for links - though that's beyond the scope of this ticket)

Thanks friends!

**Comment by bvanvugt** [comment]
This is supported in version 1.5.65 (currently live)! Thanks for the idea @smallsco :)
